docs/dev/setup.rst

```diff
@@ -156,10 +156,6 @@ You can override the following configuration options in this file.
            id: artemis_lti
            oauth-key: artemis_lti_key
            oauth-secret: <secret>    # only important for online courses on the edX platform, can typically be ignored
```

Comment:
I think you said, you can delete those variables

Reply:
Done!